src/bde_copy.cpp

```diff
@@ -1726,15 +1752,20 @@ int main( int argc, char *argv[] )
 
     if( use_gzip )
     {
-        out = gzip_data_writer::open(outfile,append,gzipbuffsize);
+      out = gzip_data_writer::open(outfile,append,gzipbuffsize);
+    }
+    else if ( output_stdout )
+    {
+      char outf[] = "-";
+      out = file_data_writer::open(outf, append);
```

Comment:
had you considered assigning "-" to the outfile variable ? Could simplify some code here and before (and after?)

Reply:
I have updated this, which does simplify it a bit.